src/katgpucbf/fgpu/recv.py

```diff
@@ -196,8 +204,7 @@ def make_stream_group(
     """
     # Reference counters to make the labels exist before the first scrape
     for pol in range(N_POLS):
-        for counter in _PER_POL_COUNTERS:
-            counter.labels(pol)
+        counters.labels(str(pol))
```

Comment:
Fair enough. However, you've added the `input_chunks` Counter to this list/set of Counters which doesn't seem to accommodate/allow for the `["pol"]` label. 
- Does it just ignore this request?
- Or rather, how does it acknowledge this "instantiation"? (to make sure the label exists before the first scrape)